charts/kube-master/templates/api.yaml

```diff
@@ -107,9 +107,14 @@ spec:
             - --authorization-mode=Node,RBAC
             - --cloud-config=/etc/kubernetes/cloudprovider/openstack.config
             - --cloud-provider=openstack
+{{- if hasPrefix .Values.version.kubernetes "v1.8" }}
```

Comment:
I don't see a point in having the *v* prefix in our values for the kubernetes version and would find it clear to just have the version number for comparison.